src/XenCert/blockunblockhbapaths-brocade

```diff
@@ -55,10 +58,10 @@ password = sys.argv[3].split(':')[2]
 portList = sys.argv[3].split(':')[3].split(',')
 sampled_portlist = portList
 
-if op == 'block': 
+if op == 'block':
     sampled_portlist = random.sample(portList, random.randint(1, len(portList) - 1))
     XenCertPrint("Blocking ports: %s" % sampled_portlist)
-    retVal=ip + ':' + username + ':' + password + ':'
+    retVal=ip + ':' + username + ':' + HIDDEN_PASSWORD + ':'
```

Comment:
It's a mistake to replace directly, because retVal is used in not only print
    XenCertPrint(retVal)
    sys.stdout.write(retVal)